packages/canari/PKGBUILD

```diff
@@ -15,8 +15,13 @@ source=("https://pypi.python.org/packages/bf/df/901531f4d2a548eb8e87f4da5f4d8b19
 sha1sums=('1ace7a39055b5245e95f520d75dab01d8181b50b')
 
 build(){
-  cd "$srcdir/canari3-$pkgver-rc1"
-  
+
```

Comment:
Could you remove those blank lines?

Reply:
Sure! I'm sorry, I resolved the conflicts with the latest master changes directly from the phone using the github interface.